[025]  Bug: Links fixed

Lucerrion opened the pull request:
025   Bug: Links fixen https://app.asana.com/0/1205072348884926/1205169427577317/f

Lucerrion committed:
footer fix like said in discord

Lucerrion committed:
Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

Lucerrion committed:
Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

# Conflicts:
#	password-generator.html

SpaTeamAcademy reviewed (changes requested):
Footer wird auf der Team Page noch nicht richtig angezeigt.

Lucerrion committed:
[025] CSS and classes added again

025 - Bug: Links fixen https://app.asana.com/0/1205072348884926/1205169427577317/f

Lucerrion committed:
[025] Merge branch '025---Bug-Links-fixen' of github.com:SpaTeamAcademy/Team_A

# Conflicts:
#	style.css
#	to-do.html
